docs/superpowers/plans/2026-04-29-system-events-failure-reason.md

```
@@ -0,0 +1,468 @@
+```
+
+Save to `migrations/postgres/V4__add_failure_reason_to_system_events.up.sql`.
+
+- [ ] **Step 2: Verify the file**
+
+```bash
+cat migrations/postgres/V4__add_failure_reason_to_system_events.up.sql
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)